courses/forms.py

```
@@ -0,0 +1,53 @@
+    )
+    course_image = CloudinaryFileField()
+
+    class Meta:
+        model = Course
+        fields = ['title', 'short_introduction', 'course_description',
+                  'requirements', 'learning_objectives', 'level',
+                  'course_image', 'preview_video_link',
+                  'upcoming', 'published', 'paid_course',
+                  'course_price']
+
+    def __init__(self, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)